tests/lac/pointer_matrix_03.cc

```diff
@@ -38,7 +38,7 @@ main()
   deallog << std::setprecision(4);
   deallog.attach(logfile);
 
-  char *name = "Matrix A";
+  char name[] = "Matrix A";
```

Comment:
the canonical way to get around the warning would have been to say
```
  const char *name = "...";
```
But it's in a test -- it makes no difference.